src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 > :nth-child(-n+4) {
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.